### ISSUE TITLE
A section on font setting in Keyman Developer is missing?

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/help.keyman.com#598]
I was not able to find a help content on how to define a font for the desktop OSK, Touch Layout and code view under https://help.keyman.com/developer/15.0/context. Can it be found else where?

**Comment by mcdurdin** [comment]
Yeah, we need a rewrite of most of the user interface help for Developer. Don't have the resources to tackle it just now though!

**Comment by darcywong00** [comment]
See also keymanapp/keyman#13679
